app/Artist.php

```diff
@@ -37,4 +43,24 @@ public function user()
     {
         return $this->catalogable->user();
     }
+
+    public function scopeFeaturable(Builder $query): Builder
+    {
+        return $query->has('profile')
```

Comment:
This query will always be `true` due to the fact that when an artist is created via the `create()` method a profile is also created as seen here https://github.com/indiehd/web-api/blob/master/app/Repositories/ArtistRepository.php#L44

Maybe its best to remove the profile rule all together, I was thinking initially that the CatelogEntity had nullable fields that we would require to have values in order to be featured. However i am now seeing that i was mistaken (or confused). Lets just eliminate this particular query unless you feel that there is a good reason (or alternative proposal) to keep this in ?

Reply:
An excellent point. I removed it and all tests still pass, which says quite enough. 👍